azure-spring-boot/pom.xml

```diff
@@ -69,6 +69,14 @@
             <artifactId>spring-security-web</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-oauth2-client</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-oauth2-core</artifactId>
```

Comment:
No need to explicitly include `spring-security-oauth2-core` as `spring-security-oauth2-client` will pull it in as a dep.

Reply:
Will remove the core.